nemo/collections/tts/losses/glow_tts_loss.py

```diff
@@ -0,0 +1,93 @@
+# MIT License
```

Comment:
Can we follow this: https://github.com/NVIDIA/NeMo/pull/988#discussion_r465936184 for copyright headers?